harvdev_utils/psycopg_functions/fb_chado_classes.py

```diff
@@ -0,0 +1,727 @@
+"""Module:: fb_chado_classes.
+
+Synopsis:
+    FlyBase chado objects (non-feature).
+
+Author(s):
+    Gil dos Santos dossantos@morgan.harvard.edu
+
+"""
+
+import logging
+import re
+import datetime
+import strict_rfc3339
+from harvdev_utils.char_conversions import sub_sup_sgml_to_html, sgml_to_unicode
+
+log = logging.getLogger(__name__)
+
+
+# Classes
+class Author(object):
```

Comment:
Not sure if this is important but these are really Pub Author objects. Reading the args this is fairly obvious so maybe I am being too picky.

Reply:
You're right. I'll fix that.